resources/js/src/checkout/blocks/components/Label.js

```diff
@@ -0,0 +1,7 @@
+export const Label = ( { item } ) => {
+	return (
+		<>
+			<div dangerouslySetInnerHTML={ { __html: item.label } } />
```

Comment:
I assume we are expecting markup within `item.label` ? Otherwise I wonder why we aren't simply passing it into the content

Reply:
yes, but there is no need to pass markup really, now we pass only data I've updated it